.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode7.3
+osx_image: xcode8.0
```

Comment:
See #30 I think it works the best that way. We made changes to `.travis.yml` that got the tests to run.